File: Week 6 - Python/sentimental-cash/cash.py
from cs50 import get_float

change = 0

while True:
    change = get_float("Change: ") * 100
    if change > 0:
        break

available = [25, 10, 5, 1]

coins = 0

for coin in available:
    times = int(change/coin)
    coins += times
    change -= (coin * times)
    if (change == 0):
        break

print(coins)

File: Week 1 - C/cash/cash.c
#include <cs50.h>
#include <stdio.h>

// get $ (change)
// return minimal number of coins to be given

int main(void)
{
    int change;
    int coinsCount = 0;
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 0);
    while (change > 0)
    {
        if (change >= 25)
        {
            change -= 25;
            coinsCount++;
        }
        else if (change >= 10)
        {
            change -= 10;
            coinsCount++;
        }
        else if (change >= 5)
        {
            change -= 5;
            coinsCount++;
        }
        else if (change >= 1)
        {
            change -= 1;
            coinsCount++;
        }
    }
    printf("%i \n", coinsCount);
}
